plugins/auto-sizes/hooks.php

```diff
@@ -159,7 +159,7 @@ function auto_sizes_improve_image_sizes_attributes( string $content ): string {
 	}
 
 	// Skips second time parsing if already processed.
-	if ( false === $processor->get_attribute( 'data-needs-sizes-update' ) ) {
+	if ( null === $processor->get_attribute( 'data-needs-sizes-update' ) ) {
```

Comment:
Per [WP_HTML_Tag_Processor::get_attribute](https://developer.wordpress.org/reference/classes/wp_html_tag_processor/get_attribute/) documentation, It return null instead of `false` when element is not presented.

Reply:
Yes, good catch.